src/main/java/gov/cabinetoffice/gap/adminbackend/dtos/application/EncryptedLastUpdatedEmailDTO.java

```diff
@@ -0,0 +1,9 @@
+package gov.cabinetoffice.gap.adminbackend.dtos.application;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+public record EncryptedLastUpdatedEmailDTO (byte[] encryptedLastUpdatedEmail) {
```

Comment:
we could probably use this DTO for other encrypted email addresses/strings. Could we give it a more generic name? `EncryptedEmailAddressDTO` ?